src/librustdoc/html/render.rs

```diff
@@ -2109,15 +2110,16 @@ fn render_assoc_item(w: &mut fmt::Formatter,
               g: &clean::Generics,
               selfty: &clean::SelfTy,
               d: &clean::FnDecl,
-              link: AssocItemLink)
+              link: &AssocItemLink)
               -> fmt::Result {
         use syntax::abi::Abi;
 
         let name = meth.name.as_ref().unwrap();
         let anchor = format!("#{}.{}", shortty(meth), name);
         let href = match link {
```

Comment:
More usual style would be to write `match *link` here and simply remove the `&` prefix on each pattern.